libnavui-speedlimit/src/main/java/com/mapbox/navigation/ui/speedlimit/ComponentInstaller.kt

```diff
@@ -0,0 +1,30 @@
+package com.mapbox.navigation.ui.speedlimit
+
+import androidx.annotation.StyleRes
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.ui.base.installer.ComponentInstaller
+import com.mapbox.navigation.ui.base.installer.Installation
+import com.mapbox.navigation.ui.speedlimit.internal.SpeedLimitComponent
+import com.mapbox.navigation.ui.speedlimit.view.MapboxSpeedLimitView
+
+/**
+ * Install components that render [MapboxSpeedLimitView].
+ *
+ * @param style definition for [MapboxSpeedLimitView]
+ * @param textAppearance for [MapboxSpeedLimitView]
+ * @param speedLimitView [MapboxSpeedLimitView]
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+fun ComponentInstaller.speedLimit(
+    @StyleRes style: Int,
+    @StyleRes textAppearance: Int,
+    speedLimitView: MapboxSpeedLimitView
+): Installation {
+    return component(
+        SpeedLimitComponent(
+            style = style,
+            textAppearance = textAppearance,
+            speedLimitView = speedLimitView
+        )
+    )
+}
```

Comment:
```suggestion
 * @param speedLimitView [MapboxSpeedLimitView]
 */
@ExperimentalPreviewMapboxNavigationAPI
fun ComponentInstaller.speedLimit(
    speedLimitView: MapboxSpeedLimitView,
    config: SpeedLimitConfig.() -> Unit = {}
): Installation {
    val componentConfig = SpeedLimitConfig().apply(config)
    return component(
        SpeedLimitComponent(
            style = componentConfig.style,
            textAppearance = componentConfig.textAppearance,
            speedLimitView = speedLimitView
        )
    )
}

@ExperimentalPreviewMapboxNavigationAPI
class SpeedLimitConfig internal constructor() {
    @StyleRes var style: Int = R.style.mapbox_speedLimit_style
    @StyleRes var textAppearance: Int = R.style.mapbox_speedLimit_textAppearance
}
```
Let's simplify this API and update it to increase its consistency with other ComponentInstaller definitions. We should encapsulate both `style` and `textAppearance` values in `SpeedLimitConfig` and initialize both with default `@StyleRes` values.